### ISSUE TITLE
Binance - code":-1106,"msg":"Parameter 'timeInForce' sent when not required.

### ISSUE THREAD

**Issue by Prossi79** [freqtrade/freqtrade#1649]
Again problems on Binance - this time no stop loss on exchange:
What does this mean? code":-1106,"msg":"Parameter 'timeInForce' sent when not required.

### Relevant code exceptions or logs:

  ```2019-03-11 09:58:23,303 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 09:58:54,594 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 09:59:25,882 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 09:59:57,185 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:00:31,403 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:01:02,706 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:01:34,014 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:02:05,322 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:02:36,603 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:03:07,906 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...
2019-03-11 10:03:39,185 - freqtrade.freqtradebot - WARNING - Error: Could not place sell order due to ExchangeError. Message: binance {"code":-1106,"msg":"Parameter 'timeInForce' sent when not required."}, retrying in 30 seconds...

  ```

**Comment by Prossi79** [comment]
just checked the data base and Binance Log - on Binance the sell order is expired and in the SQL database it is logged as closed. This should not happen right? What to do?

On telegram the status table shows no updated values too...

**Comment by xmatthias** [comment]
What's your `order_types` configuration? 

I do have a suspicion ...

**Comment by Prossi79** [comment]
order types:

"order_types": {
    "buy": "limit",
    "sell": "limit",
    "stoploss": "market",
    "stoploss_on_exchange": false,
    "stoploss_on_exchange_interval": 60
    },

order time in force:

    # Optional order time in force
    order_time_in_force = {
        'buy': 'fok',
        'sell': 'fok'
    }

I think I know your suspicion - because of the fok type? should I stay with gtc?

**Comment by xmatthias** [comment]
It's an unfortunate combination apparently noone encountered so far:
fok works fine if you don't use "market" orders.

the combination that breaks it ... #1652 should fix it, if you wanna help test it (i do so too, but it takes a while until my strategy generates a buy and corresponding sell ...)

**Comment by xmatthias** [comment]
The combination of market orders + different timeInForce should now be fixed (since merging #1652).

Feel free to reopen if it appears again after upgrading.

**Comment by OtenMoten** [comment]
Hello community,

I used the latest version and got this error:

`2019-11-30 09:52:32,084 - freqtrade - ERROR - Time in force policies are not supported for Binance yet.`
